docs/guides/enforce-active-organization.mdx

```diff
@@ -0,0 +1,208 @@
+
+Making the user a member of an organization will not make them active in that organization. The user would need to use the [`<OrganizationList />`](/docs/components/organization/organization-list) or [`<OrganizationSwitcher />`](/docs/components/organization/organization-switcher) components to select an organization, or their active organization could be set programmatically with the [`setActive()`](/docs/references/javascript/clerk/session-methods#set-active) method from the [`useOrganizationList()`](/docs/references/react/use-organization-list) component. The components in their default configuration allow the user to switch back to their Personal Workspace.
+
+## How do I know if a user is active in an organization?
+
+You can check the `orgId` value on the [Auth](docs/references/nextjs/auth-object#auth-object). If this value is `undefined` then the user is not active in an organization and is instead in their Personal Workspace.
+
+If you check the returned Auth object from [auth()](/docs/references/nextjs/auth) and the user is not active in an organization, it would like this:
+
+```json
+{
+  "sessionId": "sess_2GaMqUCB3Sc1WNAkWuNzsnYVVEy",
+  "userId": "user_2F2u1wtUyUlxKgFkKqtJNtpJJWj",
+  "claims": {
+    "azp": "http://localhost:3000",
+    "exp": 1666622607,
+    "iat": 1666622547,
+    "iss": "https://clerk.quiet.muskox-85.lcl.dev",
+    "nbf": 1666622537,
+    "sid": "sess_2GaMqUCB3Sc1WNAkWuNzsnYVVEy",
+    "sub": "user_2F2u1wtUyUlxKgFkKqtJNtpJJWj"
+  }
+}
+```
+
+## Enforcing an active organization
+
+<Steps>
+  ### Configuring Middleware
+
+  Let's start with a Middleware and that many applications could use. This example will do the following:
+
+  1. Check if the user accessing a private route is signed in. If the user is not, they are redirected to the `/sign-in` page.
+  1. Check if the user accessing a private route is signed in and if the user is then allow them to access that route.
+  1. Any use who is not accessing a public route will get access to the prublic.
+
+  ```typescript {{ filename: '/middleware.ts' }}
+  import { clerkMiddleware, createRouteMatcher } from '@clerk/nextjs/server'
+  import { NextRequest, NextResponse } from 'next/server'
+
+  const isPublicRoute = createRouteMatcher(['/', '/sign-in', '/sign-up'])
+
+  export default clerkMiddleware((auth, req: NextRequest) => {
+    // If the user isn't signed in and the route is private, redirect to sign-in
+    if (!auth().userId && !isPublicRoute(req))
+      return auth().redirectToSignIn({ returnBackUrl: req.url })
+
+    // If the user is logged in and the route is protected, let them view.
+    if (auth().userId && !isPublicRoute(req)) return NextResponse.next()
+  })
+
+  export const config = {
+    matcher: ['/((?!.*\\..*|_next).*)', '/', '/(api|trpc)(.*)'],
+  }
+  ```
```

Comment:
We don't need two separate examples, when the second example is the same code as the first with the addition of the orgId check. We can simply say something like:

```suggestion
The `orgId` value on the `Auth` object can be used to check if a user has an active organization. If this value is `undefined`, then the user is not active in an organization and is in their personal workspace.

To protect your application from users that do not have an active organization, configure your Middleware to check the user's `orgId`. 

The following example configures `clerkMiddleware()` to protect all routes except `/`, `/sign-in`, and `/sign-up`. It will handle both authentication _and_ authorization. If a user is not signed in, they will be redirected to the sign-in page. If a user does not have an active organization, they will be redirected to`/organization-selection`, which you will create in the next step.
```